exporter/collector/integrationtest/testdata/fixtures/metrics/basic_prometheus_metrics.json

```diff
@@ -51,6 +51,19 @@
                          ]
                       }
                    },
+                   {
+                     "name":"scrape_series_added",
```

Comment:
nit could you give rename this one to indicate what it's testing

Reply:
I wanted this to show that it worked for an existing metric that has valid data points, so I moved this to its own test that explains it better